lib/puppet/provider/selinux_fcontext/semanage.rb

```diff
@@ -9,13 +9,7 @@
 
   mk_resource_methods
 
-  osfamily        = Facter.value('osfamily')
-  osversion       = Facter.value('operatingsystemmajrelease')
-  operatingsystem = Facter.value('operatingsystem')
   @old_semanage = false
```

Comment:
This is now redundant. So you can also simplify it further: https://github.com/voxpupuli/puppet-selinux/pull/372